assets/css/signs.css

```diff
@@ -31,6 +31,10 @@
   padding-bottom: 6px;
 }
 
+.btn {
```

Comment:
Maybe we should add a separate class instead of altering an existing bootstrap one?

Reply:
i took inspiration from dotcom, which edits btn - but ill just rename it